tensorflow/core/tpu/tpu_api_dlsym_initializer.cc

```diff
@@ -15,23 +12,8 @@ limitations under the License.
 
 #include "tensorflow/core/tpu/tpu_api_dlsym_initializer.h"
 
-#include <dlfcn.h>
-#include <fcntl.h>
-#include <sys/mman.h>
-#include <sys/stat.h>
-#include <sys/types.h>
-#include <unistd.h>
-
-#include "tensorflow/core/platform/errors.h"
-#include "tensorflow/core/platform/status.h"
-#include "tensorflow/core/tpu/tpu_api_dlsym_set_fn.h"
-
 #if !defined(PLATFORM_GOOGLE)
-#include "tensorflow/core/platform/cloud/gcs_file_system.h"
-#include "tensorflow/core/platform/env.h"
-#include "tensorflow/core/tpu/tpu_api.h"
 #include "tensorflow/core/tpu/tpu_initializer_helper.h"
-#include "tensorflow/stream_executor/tpu/tpu_platform.h"
 #endif
 
 
```

Comment:
There's a comment directly below this line about tpu_library_loader_windows.cc. I think it's stale, so just remove it.